universal-application-tool-0.0.1/build.sbt

```diff
@@ -12,15 +12,20 @@ lazy val root = (project in file("."))
       javaJdbc,
       // JSON libraries
       "com.jayway.jsonpath" % "json-path" % "2.5.0",
-      "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % "2.10.3",
-      "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % "2.10.3",
+      "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % "2.12.5",
+      "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % "2.12.5",
+      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.12.5",
```

Comment:
question (non-blocking): were these dependency version changes necessary for the azure SDK?

Reply:
Yes, we were getting dependency conflict warnings and NoClassDefFoundErrors because the azure SDK requires at least v2.12.5

Reply:
Got it, thanks!